cmd/execution/main.go

```diff
@@ -360,8 +359,6 @@ func main() {
 				node.Storage.Blocks,
 				followerEng,
 				syncCore,
-				// Right now, only used to sync, so never poll
-				synchronization.WithPollInterval(time.Duration(0)),
```

Comment:
The execution node was not polling the blocks because the polling interval was set to 0. This caused the execution node unable to catch up after a period of downtime. 

Removing this solves the problem